specs/tables.md

```diff
@@ -104,9 +105,14 @@ NOTE: TxLog is currently Work In Progress and will be updated to include $txID i
 | $counter | $isWrite    | Memory                     | $callID  | $memoryAddress |                      |             | $value     | 0          | 0        | 0               |
 | $counter | $isWrite    | AccountStorage             |          | $address       |                      | $storageKey | $value     | $valuePrev | $txID    | $CommittedValue |
 |          |             |                            |          |             |                         |             |            |            |          |                 |
-| $counter | true        | TxLog                      | $logID   | 0           |  Address                |             | $value     | 0          | 0        | 0               |
-| $counter | true        | TxLog                      | $logID   | $topicIndex |  Topic                       |             | $value     | 0          | 0        | 0               |
-| $counter | true        | TxLog                      | $logID   | $byteIndex  |  Data                   |             | $value     | 0          | 0        | 0               |
+| $counter | true        | TxLog                     |$txID      | $logID    |  Address                 |       0     | $value     | 0          | 0        | 0               |
+| $counter | true        | TxLog                     |$txID      | $logID    |  Topic                   | $topicIndex | $value     | 0          | 0        | 0               |
+| $counter | true        | TxLog                     |$txID      | $logID    |  Data                    | $byteIndex  | $value     | 0          | 0        | 0               |
+| $counter | true        | TxLog                     |$txID      | $logID    |  Topic_Length            |       0     | $value     | 0          | 0        | 0               |
+| $counter | true        | TxLog                     |$txID      | $logID    |  Data_length             |       0     | $value     | 0          | 0        | 0               |
```

Comment:
Please update this to use CamelCase: `TopicLength` and `DataLength`

Reply:
updated in [e4fa0b9](https://github.com/appliedzkp/zkevm-specs/pull/159/commits/e4fa0b978910d0f71d63d1fb918cbab62d2a781f) and also add missing new tag `LogLength`